tests/python/requirements.txt

```diff
@@ -17,8 +17,10 @@
 coverage==7.1.0
 
 # Use to temporarily replace part of the API or mock return vaues.
-mock==2.0.0
+mock==2.0.0; python_version < '3.3'
 
 # The Shotgun Desktop does not include setuptools, which comes with pip. Make
 # sure they are installed in case the tests are run with this python.
 setuptools==65.5.1
+
+unittest2; python_version < '3'
```

Comment:
I think we can't run Python 2 tests on CI anymore. Are you running Py2 local tests just to be safe?